### PR TITLE
Release/v4.78 into development

### PR DESCRIPTION
**Pull request by blockdylanb**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by tackoman — APPROVED** [review]

**Comment by tackoman** [comment]
> ## Description (optional)
> Add a concise explanation of the changes.
> 
> ## Testing Steps (optional)
> Detail the steps required for the reviewer(s) to verify and test these changes.
> 
>

**Review by kalbroni7 — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json [hunk 1]
```diff
@@ -844,8 +844,8 @@
   "identityverification.personal.address_line2_optional": "Address Line 2 (optional)",
   "identityverification.personal.cityrequired": "City *",
   "identityverification.personal.dateofbirthrequired": "Date of Birth *",
-  "identityverification.personal.firstnamerequired": "First Name *",
```
[the remark this answers is not in the document]
Comment (kalbroni7) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: Kallai Meehan

[the remark this answers is not in the document]
Comment (kalbroni7) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: Kash77

[the remark this answers is not in the document]
Comment (kalbroni7) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: Kallai Daniel Meehan 07/04/1987
7 coora CT frankaton 3199 Vic

**Comment by tackoman** [comment]
> ## Description (optional)
> Add a concise explanation of the changes.
> 
> ## Testing Steps (optional)
> Detail the steps required for the reviewer(s) to verify and test these changes.
> 
>

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (duckman2277) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: I'm so confused right now

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json [hunk 2]
```diff
@@ -844,8 +844,8 @@
...
-  "identityverification.personal.lastnamerequired": "Last Name *",
+  "identityverification.personal.firstnamerequired": "Legal First Name *",
```
[the remark this answers is not in the document]
Comment (duckman2277) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: I wish I had the right answer but I'm clueless to as what I'm supposed to be doing

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (tackoman) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: > Tackoman73

[the remark this answers is not in the document]
Comment (tackoman) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: > Tackoman73 

Im

**Comment by tackoman** [comment]
> ## Description (optional)
> Add a concise explanation of the changes.
> 
> ## Testing Steps (optional)
> Detail the steps required for the reviewer(s) to verify and test these changes.
> 
>

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (tackoman) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: > Tackoman73

Comment (tackoman) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json:847]: Ronald McGowan

**Review by tackoman — APPROVED** [review]

**Review by kalbroni7 — APPROVED** [review]

**Review by kalbroni7 — COMMENTED** [review]
c1449643e5a7a8c7b7e79c23fcf5648905b57e47

**Pull request merged**